modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -197,9 +203,20 @@ https://anypoint.mulesoft.com/accounts/api/profile
 [IMPORTANT]
 The value in `<username>` is a literal, don't change it. The value tells the platform that you're using a token. Set your token in `<password>ACCESS_TOKEN</password>`.
 
+
+== Dependencies external to Exchange
+
+Java libraries and POM files cannot currently be published as Exchange assets.
+
+Assets that declare dependencies that are not published in Exchange are not currently supported.
```

Comment:
This is not entirely true. It only applies to Mule 4 Extensions and only to "private" dependencies that don't exist in Maven Central or MuleSoft Maven repositories

Reply:
@juancoen thank you! Is this correct?

"Java libraries and POM files cannot currently be published as Exchange assets.

A private dependency is a dependency that does not exist in Maven Central or the MuleSoft Maven repositories.

Mule 4 Extensions that declare private dependencies that are not published in Exchange are not currently supported."

Reply:
fixed by Joaquin